vscode/src/registry.json

```diff
@@ -74,6 +74,17 @@
           "path": "library/qtest"
         }
       }
+    },
+    {
+      "name": "QuantumArithmetic",
+      "description": "Quantum arithmetic algorithms",
+      "dependency": {
+        "github": {
+          "owner": "fedimser",
+          "repo": "quant-arith-re",
```

Comment:
Can you add the `refs` property also as per the other examples in this file please. If you don't current have any branches or tags to list explicitly, just have the `{ "ref": "main", "notes": "nightly, unstable" }` entry. Thanks!

Reply:
Done.